src/Services/GoogleDriveService.js

```diff
@@ -0,0 +1,63 @@
+import GDrive from 'react-native-google-drive-api-wrapper';
+import RNFS from 'react-native-fs';
+
+const mimeType = 'application/json';
+const appDataFileName = 'appData.json';
+const appDataFolder = 'appDataFolder';
+const downloadHeaderPath = RNFS.DocumentDirectoryPath + '/' + appDataFileName;
+
+export default class GoogleDriveService {
+  constructor(accessToken) {
+    this.instance = null;
+    this.acessToken = accessToken;
+
+    GDrive.setAccessToken(accessToken);
+    GDrive.init();
+  }
+
+  static getInstance(accessToken) {
+    if (
+      !GoogleDriveService.instance ||
+      GoogleDriveService.instance.accessToken != accessToken
+    ) {
+      GoogleDriveService.instance = new GoogleDriveService(accessToken);
+    }
+
+    return GoogleDriveService.instance;
+  }
+
+  deleteAppDataFile = async () => {
```

Comment:
this is very rikskily named, because it deletes _all_ files, nut just the "appdatafile"
do we need this function at all? Doesn't goolge provide some "nuke your appdata" function?

Reply:
You are right, I was using that function for development purposes. I'll delete it.